app/src/components/market/common/additional_market_data/index.tsx

```diff
@@ -101,6 +102,8 @@ interface Props extends DOMAttributes<HTMLDivElement> {
 export const AdditionalMarketData: React.FC<Props> = props => {
   const { address, arbitrator, category, curatedByDxDaoOrKleros, id, oracle, submissionIDs, title } = props
 
+  const context = useConnectedWeb3Context()
+
```

Comment:
I would suggest to destructure it... like this const {networkId}=useConnectedWeb3Context()
- So that its just networkId instead of context.networkId